src/main/webapp/resources/celJS/initCelements.js

```diff
@@ -901,25 +898,25 @@
   /**
    *  celements form validations
    */
-  var formValidations = new Hash();
+  let formValidations = new Hash();
```

Comment:
`formValidations` is never reassigned
```suggestion
  const formValidations = new Hash();
```